src/main/java/com/gregtechceu/gtceu/data/recipe/serialized/chemistry/ReactorRecipes.java

```diff
@@ -727,5 +726,161 @@ public static void init(Consumer<FinishedRecipe> provider) {
+
+        CHEMICAL_RECIPES.recipeBuilder("prussian_blue")
+                .inputItems(dust, PotassiumFerrocyanide, 3)
+                .inputFluids(Iron3Chloride.getFluid(4000))
+                .outputItems(dust, PrussianBlue, 1)
+                .outputItems(dust, RockSalt, 12)
+                .duration(500).EUt(VA[HV]).save(provider);
+
+        CHEMICAL_RECIPES.recipeBuilder("dichloroethane")
+                .inputFluids(Ethylene.getFluid(1000))
+                .inputFluids(Chlorine.getFluid(2000))
+                .notConsumableFluid(Iron3Chloride.getFluid(100))
+                .outputFluids(Dichloroethane.getFluid(1000))
+                .duration(100).EUt(VA[MV]).save(provider);
+
+        CHEMICAL_RECIPES.recipeBuilder("diethylenetriamine")
+                .inputFluids(Dichloroethane.getFluid(2000))
+                .inputFluids(Ammonia.getFluid(3000))
+                .outputFluids(Diethylenetriamine.getFluid(1000))
+                .outputFluids(HydrochloricAcid.getFluid(4000))
+                .duration(100).EUt(VA[MV]).save(provider);
+
+        CHEMICAL_RECIPES.recipeBuilder("formaldehyde")
+                .inputFluids(Methanol.getFluid(1000))
+                .inputFluids(Oxygen.getFluid(1000))
+                .outputFluids(Formaldehyde.getFluid(1000))
+                .outputFluids(Water.getFluid(4000))
+                .duration(100).EUt(VA[MV]).save(provider);
+
+        CHEMICAL_RECIPES.recipeBuilder("glycolonitrile")
+                .inputFluids(Formaldehyde.getFluid(1000))
+                .inputFluids(HydrogenCyanide.getFluid(1000))
+                .outputFluids(Glycolonitrile.getFluid(1000))
+                .outputFluids(Water.getFluid(4000))
+                .duration(100).EUt(VA[MV]).save(provider);
+
+        CHEMICAL_RECIPES.recipeBuilder("diethylenetriamine_pentaacetonitrile")
+                .inputFluids(Diethylenetriamine.getFluid(1000))
+                .inputFluids(Glycolonitrile.getFluid(5000))
+                .outputFluids(DiethylenetriaminePentaacetonitrile.getFluid(1000))
+                .outputFluids(Water.getFluid(5000))
+                .duration(100).EUt(VA[HV]).save(provider);
+
+        CHEMICAL_RECIPES.recipeBuilder("diethylenetriamine_pentaacetonitrile")
+                .inputFluids(Diethylenetriamine.getFluid(1000))
+                .inputFluids(Glycolonitrile.getFluid(5000))
+                .outputFluids(DiethylenetriaminePentaacetonitrile.getFluid(1000))
+                .outputFluids(Water.getFluid(5000))
+                .duration(100).EUt(VA[HV]).save(provider);
+
+        LARGE_CHEMICAL_RECIPES.recipeBuilder("diethylenetriaminepentaacetic_acid")
+                .inputItems(dust, SodiumHydroxide, 5)
+                .inputFluids(DiethylenetriaminePentaacetonitrile.getFluid(1000))
+                .inputFluids(Water.getFluid(5000))
+                .inputFluids(Hydrogen.getFluid(5000))
+                .outputItems(dust, DiethylenetriaminepentaaceticAcid, 1)
```

Comment:
this chain is chemical reactor hell, maybe see if it can be broken up into other machines?